list_constraints.tf

```diff
@@ -34,10 +34,10 @@ resource "google_organization_policy" "org_policy_list_allow_all" {
   Folder policy, allow all (list constraint)
  *****************************************/
 resource "google_folder_organization_policy" "folder_policy_list_allow_all" {
-  count = "${local.folder && local.list_policy && local.enforce == "false" && var.deny_list_length == 0 ? 1 : 0}"
+  count = local.folder && local.list_policy && local.enforce == false && var.deny_list_length == 0 ? 1 : 0
```

Comment:
Please use "not" operator

Reply:
local.enforce can be null, true or false. boolean evaluation of local.enforce fails when it's null